src/cli/commands/dev.ts

```diff
@@ -12,6 +12,17 @@ import { createStartModuleContent } from '../../runtime/start'
 import * as Reflection from '../../lib/reflection'
 import { simpleDebounce } from '../../lib/utils'
 
+/**
+ * 1. Unexpected crashes -> process.exit()
+ * 2. Unexpected errors -> throws
+ * 3. Hanging process
+ */
+
+/**
+* - Main process (runs nexus dev)
+* - Dev mode/Sandbox process (runs everything (reflection + watcher))
+* - App process (runs the user's app)
+*/
```

Comment:
Thinking we can move this into an [arch doc section](https://github.com/graphql-nexus/nexus/blob/master/docs/architecture.md)? Maybe the viz I created in the issue could be copied over (which has a bit more detail).

Reply:
Oooops, forgot to remove it!